lib/LMSManagers/LMSHelpdeskManager.php

```diff
@@ -1226,4 +1226,16 @@ public function MarkTicketAsUnread($ticketid) {
 		return $this->db->Execute('DELETE FROM rtticketlastview WHERE ticketid = ? AND userid = ?',
 			array($ticketid, Auth::GetCurrentUser()));
 	}
+
+	public function GetIndicatorsStats() {
+		$cuser = Auth::GetCurrentUser();
+
+		$this.critical->db->Execute('SELECT count(id) FROM rttickets WHERE priority='.RT_PRIORITY_CRITICAL.' AND (status !='.RT_RESOLVED);
```

Comment:
tu jest coś nie tak - o to również czepia się Travis.

Reply:
nieaktualne